### PR TITLE
MAUI - 870300 - Update User Guide for Grouping and Summaries

### PR DESCRIPTION
**Pull request by AbhishekSF4201**
##Description##
1.Completed changes in summaries.
2. completed changes in grouping

### CODE REVIEW

File: MAUI/DataGrid/overview.md [hunk 1]
```diff
@@ -39,8 +39,12 @@ The .NET MAUI DataGrid control is used to display and manipulate data in a tabul
 
 * **Styling** - Customize the appearance of cells and headers. Conditional styling is supported.
 
+* **Paging** - Paging provides built-in options to page data on demand when dealing with large volumes of data.
```
Comment (ashok-kuvaraja) [inline MAUI/DataGrid/overview.md:42]: * **Paging** - Support for loading data in segments when dealing with large volumes of data.

**Review by ashok-kuvaraja — CHANGES_REQUESTED** [review]
Modified the changes.

File: MAUI/DataGrid/overview.md (hunk 1 above)
Comment (AbhishekSF4201) [reply MAUI/DataGrid/overview.md]: Changed the content.

**Review by ashok-kuvaraja — APPROVED** [review]
The changes are fine.

**Pull request merged**